src/server.ts

```diff
@@ -39,6 +39,7 @@ export class ProxyServer extends EventEmitter {
 
     this.options = options || {};
     this.options.prependPath = options.prependPath !== false;
+    this.options.normalizeUrl = options.normalizeUrl !== false;
```

Comment:
We probably don't need an option if behavior is correct

Reply:
You've right!